xmpp/auth.py

```diff
@@ -19,28 +19,27 @@
 Can be used both for client and transport authentication.
 """
 
-import sys
 from .protocol import *
 from .client import PlugIn
 import base64,random,re
 from . import dispatcher
 from hashlib import md5,sha1
+from six import ensure_str,ensure_binary
 
-IS_PYTHON3 = sys.version_info >= (3, 0, 0)
+CHARSET_ENCODING='utf-8'
```

Comment:
(no action required) nit: the default encoding for `ensure_str` and `ensure_binary` is `utf-8` so this is not strictly required for those calls (although maybe nice to be explicit).